src/interface/src/app/shared/form-message-box/form-message-box.component.html

```diff
@@ -0,0 +1,10 @@
+<div *ngIf="message" class="message-box" [ngClass]="getBoxClass()">
+  <div class="title">
+    <img [src]="getMessageIcon()" alt="message-icon" class="message-icon" />
+    <p class="message-title">{{ title }}</p>
+  </div>
+  <div class="message-details">
+    {{ message }}
+    <ng-content select="[added-content]"></ng-content>
```

Comment:
if you only have one `ng-content` like on this component, there is no need to name it/ use `select`